docs/quest_concept.md

```diff
@@ -0,0 +1,17 @@
+## Quest concept
+
+### General description
+The quest presents a sequence of questions and instructions for movement (possibly not explicit). The process of passing consists in answering questions and, possibly, moving around the locations. Users earn points as they progress. At the discretion of the author, the quest may be limited in time.
+
+### Quest start
+The user, using the bot, selects the desired quest and starts its passage. After that, a welcome message is sent to him, and then the first question.
+
+### Along the quest
+After receiving an answer, the bot either sends the next question (in this case, the scores may change in any direction or not change at all), or in case of an unsatisfactory answer, repeats the current one (possibly with the removal of points), or sends an indication of movement to a specific place (in this case the quest continues after reaching this point, at the discretion of the author, he can add the time the point is active, for example, night hours). The next question or indication may depend on the answer to the current question (if the author has provided for branching of the quest).
```

Comment:
Try to reformulate, the essence of the claim is not clear.

Reply:
The branches, the types of questions/answers and how they define transitions are the most controversial points, because of which we are writing this document. I think, you should write special paragraph about it and describe very precise:
* What types of questions do we support?
* What is an answer to the question?
* Is it possible that question doesn't have answers?
* Must all answers define transitions between branchs (or places in the same branch)?
* Are there transitions without questions and answers? If yes, then describe the mechanism of transition (user point of view).
I suppose there are any other moments I missed, so I ask other teammates to compelte my points.